packages/pwa-kit-react-sdk/src/ssr/universal/contexts/index.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useEffect, useRef} from 'react'
+import PropTypes from 'prop-types'
+import {useLocation} from 'react-router-dom'
+import DeviceContext from '../device-context'
+
+const ExpressContext = React.createContext()
+
+const CorrelationIdContext = React.createContext()
+
+const CorrelationIdProvider = ({children, correlationId, resetOnPageChange = true}) => {
```

Comment:
Again, please add some jsdoc style block comments

We should probably used the `@private` indicator since we are only using it internally in react-rendering and main. This will allow use to make changes without having to worry about notifying others. On the other hand the hook is not private and the hoc is also not private.